Library/Homebrew/yard/templates/default/docstring/html/private.erb

```diff
@@ -0,0 +1,5 @@
+<p class="note private">
+  <strong>This <%= object.type %> is part of a private API.</strong>
+  This <%= object.type %> may only be used in the <a href="https://github.com/Homebrew/brew">Homebrew/brew</a> repository.
```

Comment:
```suggestion
  This <%= object.type %> should only be used in the <a href="https://github.com/Homebrew/brew">Homebrew/brew</a> repository.
```

Reply:
Why?

Reply:
Should is stronger and used more often in RFCs. I don't feel strongly, though.